pdns/recursordist/docs/changelog/4.0.rst

```diff
@@ -127,7 +127,7 @@ Improvements
 - `commit bcce047bc <https://github.com/PowerDNS/pdns/commit/bcce047bc>`__:
    e.root-servers.net has IPv6 now (phonedph1)
 - `commit cef8ec7c2 <https://github.com/PowerDNS/pdns/commit/cef8ec7c2>`__:
-   hello decaf signers (ED25519 and ED448) Testing algorithm 15: 'Decaf ED25519' ->'Decaf ED25519' -> 'Decaf ED25519' Signature & verify ok, signature 68usec, verify 93usec Testing algorithm 16: 'Decaf ED448' ->'Decaf ED448' -> 'Decaf ED448' Signature & verify ok, signature 163usec, verify 252usec (Kees Monshouwer)
+   hello decaf signers (ED25519 and ED448) Testing algorithm 15: 'Decaf ED25519' ->'Decaf ED25519' -> 'Decaf ED25519' Signature & verify ok, signature 68us, verify 93us Testing algorithm 16: 'Decaf ED448' ->'Decaf ED448' -> 'Decaf ED448' Signature & verify ok, signature 163us, verify 252us (Kees Monshouwer)
```

Comment:
These are verbatim quotes from program output, please leave as-is.

Reply:
I was wondering about that.